dockerfile_parse/util.py

```diff
@@ -188,3 +188,113 @@ def remove_nonescaped_quotes(string):
     string = remove_quotes(string)
     string = string.replace('\ ', ' ')
     return string.replace("\\s", "'").replace('\\d', '"')  # restore
+
+
+def split_tuple(text):
+    text_split = text.split('=')
```

Comment:
What about if a value contains '=', e.g. foo=bar=baz? Should this be `split('=', 1)` (like the previous code) or is it intentional the way it is?